qoqo_qasm/doc/conf.py

```diff
@@ -46,14 +45,15 @@
               'sphinx.ext.viewcode',
               'sphinx.ext.napoleon',
               'sphinx.ext.autosummary',
-              'nbsphinx', ]
+              'myst_parser',
+              'nbsphinx']
 # automatically use sphinx-autogen
 autosummary_generate = True
 autosummary_imported_members = True
-# define mock imports for packages that are difficult to handle / install
+# define qasm imports for packages that are difficult to handle / install
 alist = []
-alist.extend(['hqsbase', 'calculatorqore'])
-autodoc_mock_imports = alist
+alist.extend(['qoqo_calculator_pyo3'])
```

Comment:
actually that was not due to qoqo_mock, but sphinx itself is mocking an import, but we don't need to mock something here